### PR TITLE
M sc1 61 19

### PR DESCRIPTION
**Pull request by sergey-mario**

### CODE REVIEW

File: smaretski/02_HTML/task_01_table.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>  
+<HTML> 
+    <HEAD>
+        <META charset="utf-8">
+        <TITLE>Прогноз погоды</TITLE> 
+    </HEAD>
+    <BODY> 
+        <center><H1>Прогноз погоды</H1></center>
```
Comment (pluhin) [inline smaretski/02_HTML/task_01_table.html]: do not use center tag - it is weeeeery old, (use align attribute, at least)

File: smaretski/02_HTML/task_01_link_ img.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        <p>Теперь давайте рассмотрим пример программирования цикла. Запустите текстовый редактор и
+        наберите следующую программу:</p>
+        
+        <pre>#!/usr/bin/env python
+for cntr in range(0,10):
+    print cntr</pre>
+        <p><i>Не забудьте поставить символ табуляции перед строкой «print cntr»</i>. Это важно. В Python для
+        указания на блоки кода используются не круглые «(» и фигурные «{» скобки как в других языках
+        программирования, а отступы.</p>
+        <center><table width="100%" border="0"</table></center>
```
Comment (pluhin) [inline smaretski/02_HTML/task_01_link_ img.html]: remove center

File: smaretski/02_HTML/task_01_link_ img.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        <tr>
+        <th align="left"><a href="empty_page.html"><img src="prev.png" alt="Предыдущая страница" title="Переход на предыдущую страницу"></a></th>
+        <th><a href="#"><img src="home.png" alt="Переход к основному меню" title="Основное меню"></a></th>
+        <th align="right"><a href="task_02_link_img.html"><img src="next.png" alt="Следующая страница" title="Переход на следующую страницу"></a></th>
+        </tr>
```
Comment (pluhin) [inline smaretski/02_HTML/task_01_link_ img.html:63]: You did not close table

**Review by pluhin — CHANGES_REQUESTED** [review]

File: smaretski/02_HTML/task_01_table.html (hunk 1 above)
Comment (sergey-mario) [reply smaretski/02_HTML/task_01_table.html]: Fixed. Used align

File: smaretski/02_HTML/task_01_link_ img.html (hunk 2 above)
Comment (sergey-mario) [reply smaretski/02_HTML/task_01_link_ img.html]: Fixed. Used align

File: smaretski/02_HTML/task_01_link_ img.html (hunk 3 above)
Comment (sergey-mario) [reply smaretski/02_HTML/task_01_link_ img.html:63]: Fixed. Add </table>

**Review by pluhin — CHANGES_REQUESTED** [review]
You have conflicts, could you resolve them

**Comment by sergey-mario** [comment]
> You have conflicts, could you resolve them



> I did rebase. But I can't see my files in our branch "m-sc1-61-19"

**Pull request closed without merging**